snippets/fsh-snippets.json

```diff
@@ -114,13 +137,31 @@
 		],
 		"description": "Create a FSH Invariant"
 	},
-
 	"FSH RuleSet": {
-		"prefix": ["RS", "rs", "rule", "Rule"],
+		"prefix": [
+			"RS",
+			"rs",
+			"rule",
+			"Rule"
+		],
 		"body": [
 			"RuleSet: $1",
 			"* $0"
 		],
 		"description": "Create a FSH RuleSet"
+	},
+	"Slicing rules": {
+		"prefix": [
+			"^slicing"
+		],
+		"body": [
+			"^slicing.discriminator.type = ${1|#value,#exists,#pattern,#type,#profile|}",
+			"${TM_CURRENT_LINE/\\s*([^\\^]+).*/$1/}^slicing.discriminator.path = \"$2\"",
+			"${TM_CURRENT_LINE/\\s*([^\\^]+).*/$1/}^slicing.rules = ${3|#open,#closed,#openAtEnd|}",
+			"${TM_CURRENT_LINE/\\s*([^\\^]+).*/$1/}^slicing.description = \"$4\"",
+			"${TM_CURRENT_LINE/\\s*([^\\^]+).*/$1/}^slicing.ordered = ${5|true,false|}",
```

Comment:
Can we change the order of this so `false` comes first?  I'd like `false` to be the default (first selected) value since `true` is extremely rare.